app/Services/Media.php

```diff
@@ -75,7 +75,8 @@ public function sync($path = null, $tags = [], $force = false, SyncMedia $syncCo
         foreach ($this->gatherFiles($path) as $file) {
             $file = new File($file, $getID3);
 
-            $song = $file->sync($this->tags, $force);
+            $failReason = '';
+            $song = $file->sync($this->tags, $force, $failReason);
```

Comment:
OK I think I get why you pass by reference now. This does look like a code smell to me tbh.